Dockerfile

```diff
@@ -56,11 +59,19 @@ RUN apt-get update && apt-get -t stretch install -y \
         libmaxminddb-dev                 \
     && rm -rf /var/lib/apt/lists/*
 
+# IMPORTANT: When editing one of these lists below, make sure to also update
+# `Dockerfile.deploy`.
+
 # Install `file` and `libmagic` from the `buster` repositories for an up-to-date
 # file-detection.
 RUN apt-get update && apt-get -t buster install -y \
-       file \
-       libmagic-dev \
+        file \
+        libmagic-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install a recent libgit2-dev version...
+RUN apt-get update && apt-get -t buster-backports install -y \
```

Comment:
are these additional `apt-get update`s necessary when we just did it a few lines above?

Reply:
Yes because we clear the cache with `rm -rf /var/lib/apt/lists/*`